l10n_hr_account_base/models/account_move.py

```diff
@@ -90,6 +90,13 @@ def _auto_init(self):
                 WHERE (state = 'posted' AND name != '/' AND move_type IN ('out_invoice', 'out_refund'));
             """)
 
+    @api.depends('move_type', 'company_id')
+    def _compute_is_ref_required(self):
+        for rec in self:
+            rec.l10n_hr_is_ref_required = False
```

Comment:
@brunoFilipGithub 
probaj ovako (svaki postavljanje vrijednosti zove write tako da uvijek izracuna finalnu vrijednost pa tak ond na kraju na rec.polje postavi vrijednost. ako azuriras vise polja, tada iskoristi rec.update(polje1=val, polje2=val,...)
```suggestion
            rec.l10n_hr_is_ref_required = rec.move_type == 'in_invoice' and rec.company_id.country_id.code == 'HR' and True or False
```